rpaas/storage.py

```diff
@@ -238,3 +239,27 @@ def find_le_certificates(self, query):
             certificate["name"] = certificate["_id"]
             del certificate["_id"]
             yield certificate
+
+    def store_acl_network(self, name, src, dst):
```

Comment:
Weren't we planning on eventually removing MongoDB and using only Consul as a storage? What do you think about start adding new objects only to Consul?